ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/Transaction.java

```diff
@@ -753,6 +768,7 @@ private static Bytes32 computeSenderRecoveryHash(
         preimage = frontierPreimage(nonce, gasPrice, gasLimit, to, value, payload, chainId);
         break;
       case EIP1559:
+      case BLOB:
```

Comment:
Doesn't this new type requiere to calculate the pre-image in a different way than 1559?

Reply:
correct it does, that will be provided by another pr. defining at least the case was necessary for unit tests to pass